app/controllers/objects_controller.rb

```diff
@@ -70,8 +63,6 @@ def checksums
       end
       format.any { render status: :not_acceptable, plain: 'Format not acceptable' }
     end
-  rescue Moab::InvalidSuriSyntaxError => e
-    render(plain: "400 Bad Request: #{e}", status: :bad_request)
```

Comment:
this is comical b/c we went to some trouble to get this to come from Moab. hahahaha